cluster-scope/overlays/nerc-ocp-prod/clusterrolebindings/xdmod-reader-xdmod-permissions.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: xdmod-reader-xdmod-permissions
+subjects:
+- kind: ServiceAccount
+  name: xdmod-reader
+  namespace: xdmod
```

Comment:
@tzumainn I need to know what namespace are we doing things in so we can create one.

Reply:
@naved001 'xdmod' sounds good to me!